tests/PersistSpec.hs

```diff
@@ -26,3 +38,35 @@ regressSQLite = it "SQLite successfully closes" $ do
     _db <$> readMVar mv
   SQLite.closeSQLite db `shouldReturn` Right ()
   removeFile f
+
+dbScript :: FilePath
+dbScript = "tests/pact/db.repl"
+
+simpleTableMungerTest :: Spec
+simpleTableMungerTest = runMungerTest simpleTableMunger
+      [ "[USER_mungeModule_mungeTable_DATA]"
+      , "[USER_mungeNamespace.mungeModule_mungeTable_DATA]"
+      ]
+
+
+ucaseMungerTest :: Spec
+ucaseMungerTest = runMungerTest ucaseEncodeTableMunger
+      [ "[USER_mungem:odule.munget:able_DATA]"
```

Comment:
Did we maybe want to use `:m` rather than `m:`, just for better readability when looking at the native SQLite tables?

Reply:
It is more readable ... my only concern is at least in Checkpointer where there's no prefix, it means uppercase table names would start with `:`. I guess those are uncommon in lisp however ...